.meta/template/types.lua

```diff
@@ -521,6 +538,7 @@ do
 	---@field class string 🔒 "Vehicle"
 	---@field data table A Lua table which persists throughout the lifespan of this object.
 	---@field type VehicleType 💾
+	---@field despawnTime integer Ticks remaining until removal.
```

Comment:
There is no such property.

Reply:
This works for me, -1 is the default and disables the despawntime.